cmd/state/internal/cmdtree/ppm.go

```diff
@@ -1,135 +1,134 @@
 package cmdtree
 
 import (
-	"fmt"
-	"os"
-	"os/exec"
 	"strings"
 
-	"github.com/ActiveState/cli/internal/analytics"
 	"github.com/ActiveState/cli/internal/captain"
-	"github.com/ActiveState/cli/internal/errs"
 	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
 	"github.com/ActiveState/cli/internal/primer"
-	"github.com/ActiveState/cli/pkg/projectfile"
+	"github.com/ActiveState/cli/internal/runners/ppm"
 )
 
-func printSuggestion(prime *primer.Values, ppmIntent, newCommand, docLink string) error {
-	prime.Output().Print(locale.Tr("ppm_print_suggestion", ppmIntent, newCommand, docLink))
+func printSuggestion(out output.Outputer, ppmIntent, newCommand, docLink string) error {
+	out.Print(locale.Tr("ppm_print_suggestion", ppmIntent, newCommand, docLink))
 	return nil
 }
 
-func printDefault(prime *primer.Values) error {
-	prime.Output().Print(strings.TrimSpace(locale.T("ppm_header_message")))
+func printDefault(out output.Outputer) error {
+	out.Print(strings.TrimSpace(locale.T("ppm_header_message")))
 	return nil
 }
 
-func printMain(prime *primer.Values) error {
-	prime.Output().Print(locale.T("ppm_print_main"))
+func printMain(out output.Outputer) error {
+	out.Print(locale.T("ppm_print_main"))
```

Comment:
Let's move this to the runner also, it looks out of place seeing this in the cmdtree.